### ISSUE TITLE
sound._Player looping does not function

### ISSUE THREAD

**Issue by zeziba** [pvcraven/arcade#245]
## Bug Report
Pyglet the underlying framework for the Player has a method called ```on_eos```, this function is not implemented nor does it throw the not implemented error when used to control the functionality of the player. In the newest version of pyglet 1.4.0a1 this is remedied with the appropriate code added.

### Actual behavior:
Looping does not occur

### Expected behavior:
Looping of music tracks should occur

### Steps to reproduce/example code:
The following happens

```python
import arcade
import os
path = "c:\\" + os.path.join(*arcade.__file__.split("\\")[1:-1], "examples", "sounds")
player = arcade.Player()
player.load_dir(path)
player.looping(True)
player.play(player.music[0])
arcade.run()
```

After performing the above it plays the sound once and fails to repeat.

**Comment by zeziba** [comment]
I fixed the issue with looping on windows, if someone could check on mac os and linux that would be very helpful.

Use the above code to easily test it out.

**Comment by pvcraven** [comment]
Closed with 2.0 release.
